pycqed/analysis_v2/timedomain_analysis.py

```diff
@@ -501,7 +509,7 @@ def cal_states_analysis(self):
             {qbn: '' for qbn in self.qb_names})
         for qbn in self.qb_names:
             cal_states_dict = self.cal_states_dict_for_rotation[qbn]
-            if len(cal_states_dict) not in [2, 3]:
+            if len(cal_states_dict) > 3:
```

Comment:
Is this change to allow len 0? Shouldn't it then be
```
if len(cal_states_dict) not in [0, 2, 3]:
```
and the error message should be adapted accordingly?

Reply:
I want to allow up to 3 cal states per qubit.